metron-deployment/roles/epel/tasks/main.yml

```diff
@@ -16,6 +16,4 @@
 #
 ---
 - name: Install EPEL repository
-  yum: name=epel-release update_cache=yes
-
-
+  yum: name=epel-release
```

Comment:
There is no need to force a cache update here.  This role gets run repetitively and forcing a cache update just slows us down.